easytest/noncompliant

```diff
@@ -0,0 +1,8 @@
+"""
+add some non pep8 compliant code to test stickler-ci
+"""
+
+
+def foo(a):      
+ x=5.
+ b    =    [    'Hello'    +    'World']
```

Comment:
@bulli92 I think that the reason this is not working is that the file extension is not `.py`

I done a similar experiment on a toy repo of mine.

See this [pull-request](https://github.com/bjlittle/stickler-test/pull/4) which adds a non-compliant python file `bad.py` and stickler-ci correctly complains about it. Hoo-rah!

Now, see this [pull-request](https://github.com/bjlittle/stickler-test/pull/5) which adds the **same** non-compliant python file but it's called `stuff` instead of `stuff.py` ... and stickler-ci **doesn't** raise an exception to it